packages/react/react/src/hooks/useForceUpdate.ts

```diff
@@ -7,10 +7,10 @@ const updater = (num: number): number => (num + 1) % 1_000_000;
  * @description
  * 반환된 함수를 실행 시 강제로 리렌더가 실행됩니다.
  *
- * @note useReduce가 state보다 조금 더 저렴(?) 하다고 합니다. https://github.com/streamich/react-use/pull/837
+ * @note useReduce가 state보다 가볍다는 의견이 있습니다. https://github.com/streamich/react-use/pull/837
```

Comment:
How about using `@link`
[jsdoc - @link](https://jsdoc.app/tags-inline-link.html) 

```suggestion
 * @note {@link https://github.com/streamich/react-use/pull/837 useReduce가 state보다 가볍다는 의견이 있습니다.}
```

Reply:
Thank you for such a great opinion. reflected it.